sympy/combinatorics/tests/test_pc_groups.py

```diff
@@ -73,18 +73,18 @@ def test_exponent_vector():
 
 
 def test_induced_pcgs():
-    G = SymmetricGroup(9).sylow_subgroup(3)
-    PcGroup = G.polycyclic_group()
-    collector = PcGroup.collector
-    gens = [G[0], G[1]]
-    ipcgs = collector.induced_pcgs(gens)
-    order = [gen.order() for gen in ipcgs]
-    assert order == [3, 3]
-
-    G = SymmetricGroup(20).sylow_subgroup(2)
-    PcGroup = G.polycyclic_group()
-    collector = PcGroup.collector
-    gens = [G[0], G[1], G[2], G[3]]
-    ipcgs = collector.induced_pcgs(gens)
-    order = [gen.order() for gen in ipcgs]
-    assert order ==[2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2]
+    G = [SymmetricGroup(9).sylow_subgroup(3), SymmetricGroup(20).sylow_subgroup(2), AlternatingGroup(4),
+    DihedralGroup(4), SymmetricGroup(3), SymmetricGroup(4)]
```

Comment:
Can you think of few more examples? So, that we'll be sure that its working as expected.